packages/@ama-sdk/schematics/schematics/typescript/core/openapi-codegen-typescript/src/main/java/com/amadeus/codegen/ts/AbstractTypeScriptClientCodegen.java

```diff
@@ -336,7 +322,6 @@ public String getTypeDeclaration(Schema p) {
     return super.getTypeDeclaration(p);
   }
 
-  //  TODO check logic here, the way of handling stuff changed :(
   @Override
   public String getSchemaType(Schema schema) {
     // TODO test this use case as we can get much more type
```

Comment:
Is it done?
Does this comment still meaningful

Reply:
I checked it, I tested it without the override and it generates types we do not support. The mechanism is a safeguard to make sure we keep control over the types generated.